### ISSUE TITLE
Adding SSL when with AccessList results in bad Nginx config

### ISSUE THREAD

**Issue by cmitz** [jc21/nginx-proxy-manager#466]
### Checklist
- [x] Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image?
- [x] Are you sure you're not using someone else's docker image?
- [Yes, but with Access List] If having problems with Lets Encrypt, have you made absolutely sure your site is accessible from outside of your network?

### Describe the bug
_A clear and concise description of what the bug is._  
When adding SSL with LetsEncrypt to a domain that has an AccessList, the system provides a LE Certificate but generates an invalid new config.

_What version of Nginx Proxy Manager is reported on the login page?_
**v2.3.1**

**To Reproduce**
1. Make Access List with 1 allowed IP range
    1. Go to _Access Lists_
    1. Click on _Add Access List_
    1. Fill in a name
    1. Go to tab _Access_
    1. Fill in `192.168.1.0/24`
    1. Click Save
1. Make Domain
    1. Go to _Hosts_ -> _Proxy Hosts_
    1. Click on _Add Proxy Host_
    1. Add a name, IP and port
    1. Select previously created access list
    1. Click _Save_
1. Add SSL
   1. Click on the three dots -> _Edit_
   1. Go to tab _SSL_
   1. In _SSL Certificate_ select "Request a new certificate with Let's Encrypt"
   1. Click _Save_
1. See alert "Internal Error"
1. The status of the proxy host is now "unkown"

To fix this, go back to open the Access List (edit) and just press _Save_ without changing it. _Disable_ and _enable_ the proxy host again and all is well.

This happens also when:
- Access Lists contained Basic Auth, no IP ranges
- Proxy was newly created, not saved yes, before SSL with LE option was selected. After all fields were filled in _Save_ was pressed

### Expected behavior
I expected the dialog to close with no errors.

### Screenshots
Screenshots with proxy 6:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/5594436/85176443-19fd3400-b27a-11ea-9435-a5556be5fc48.png">
<img width="733" alt="image" src="https://user-images.githubusercontent.com/5594436/85176474-2e413100-b27a-11ea-9b28-178dd41a1b08.png">

### Operating System
I use NginxProxyManager in a Docker setup using UnRaid (Ubuntu).

### Additional context
The auto-generated, slightly censored config after adding the LE certificate:

```nginx
# ------------------------------------------------------------

server {
  set $forward_scheme http;
  set $server         "192.168.1.42";
  set $port           8080;

  listen 80;
#listen [::]:80;

  server_name sub.domain.nl;

  access_log /data/logs/proxy_host-5.log proxy;

  location / {

    # Access Rules
    deny all;

    # Access checks must...
    ;

    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $http_connection;
    proxy_http_version 1.1;

    # Proxy!
    include conf.d/include/proxy.conf;
  }


  # Custom
  include /data/nginx/custom/server_proxy[.]conf;
}
```

The config after opening and saving the AccessList again:

```nginx
# ------------------------------------------------------------

server {
  set $forward_scheme http;
  set $server         "192.168.1.42";
  set $port           8080;

  listen 80;
#listen [::]:80;

  server_name sub.domain.nl;

  access_log /data/logs/proxy_host-5.log proxy;

  location / {

    # Access Rules
    allow 192.168.1.0/24;
    allow 192.168.33.0/24;
    deny all;

    # Access checks must...
    satisfy any;

    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $http_connection;
    proxy_http_version 1.1;

    # Proxy!
    include conf.d/include/proxy.conf;
  }

  # Custom
  include /data/nginx/custom/server_proxy[.]conf;
}
```

**Comment by R0GGER** [comment]
I am having exact same issues.. Can somebody patch this one?

**Comment by hlidar85** [comment]
I have the same problem here

**Comment by Baukebrenninkmeijer** [comment]
Same problem!

**Comment by alex20465** [comment]
same problem

**Comment by cmitz** [comment]
This was fixed in 2.6.1 I think. https://github.com/jc21/nginx-proxy-manager/releases/tag/v2.6.1

At least for me it was :)
